modules/graftorio/require.lua

```diff
@@ -0,0 +1,23 @@
+local Commands = require("expcore.commands")
+local config = require("config.graftorio")
+
+local general = require("modules.graftorio.general")
```

Comment:
`general` needs a similar test which forcestats has.
```suggestion
local general; if config.modules.general then
  general = require("modules.graftorio.general")
end
```